atlp-rwanda/Phantom-Backend-Elites
# Organizing code structure

## What does this PR do?
​
- Refactors the folder structure.
- Removes duplicate files.
​
## Description of Task to be completed?
​
- Removed configs folder and remains config.
- Removed middlewares folder and remain middleware.
- Remove app.js file as a server and remain server.js
- Remove the publics folder and remain public.
​
## How should this be manually tested?
​
- clone the repository, then run `npm i` to install all depandencies.
- start the server with `npm start`
- run `npm test` to check all the tests are passing.

​
## What are the relevant Trello board cards?
​
​
## Did you test this in postman
​
- Yes, go to this http://localhost:3000/ and make a GET request.
